FluentStorage/Blobs/Files/DiskDirectoryBlobStorage.cs

```diff
@@ -96,10 +98,10 @@ internal class DiskDirectoryBlobStorage : IBlobStorage {
 				var di = new DirectoryInfo(fullPath);
 
 				var blob = new Blob(relPath, BlobItemKind.Folder);
-				blob.LastModificationTime = di.CreationTimeUtc;
+				blob.LastModificationTime = di.LastWriteTime;
+				blob.CreatedTime = di.CreationTime;
 				blob.TryAddProperties(
-				   "LastAccessTimeUtc", di.LastAccessTimeUtc.ToString(),
-				   "LastWriteTimeUtc", di.LastWriteTimeUtc.ToString(),
+				   "LastAccessTime", di.LastAccessTimeUtc.ToString("u"),
```

Comment:
can you explain what this does please?

Reply:
The same idea as before. This code is for the directory, the previous for files.

Reply:
Also corrected the label "LastAccessTimeUtc" in the second commit in this pull request